pkg/kritis/review/validating_transport.go

```diff
@@ -58,11 +60,16 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 		return nil, err
 	}
 	for _, a := range attestations {
-		if err = host.VerifyAttestationSignature(keys[a.KeyID], a.Signature); err != nil {
-			glog.Errorf("Could not find or verify attestation for attestor %s: %s", a.KeyID, err.Error())
-		} else {
-			out = append(out, attestation.ValidatedAttestation{AttestorName: avt.Attestor.Name, Image: image})
+		decoded_sig, err := base64.StdEncoding.DecodeString(a.Signature)
+		if err != nil {
+			glog.Infof("Cannot base64 decode signature: %v", err)
```

Comment:
should this be of severity `Error`?

Reply:
I think it shouldn't.  Anyone can create any kind of Occurrence, and there's nothing we can do about it.

Reply:
good point, sgtm